src/internal.c

```diff
@@ -9718,6 +9718,20 @@ static int GetDtls13RecordHeader(WOLFSSL* ssl, const byte* input,
             return SEQUENCE_ERROR;
     }
 
+    ret = Dtls13GetUnifiedReaderSize(
+        *(input+*inOutIdx), &ssl->dtls13CurRlLength);
+    if (ret != 0)
+        return ret;
```

Comment:
There needs to be a check here to make sure that `*(input+*inOutIdx)` is at least 1 byte in length. I don't see if that is happening anywhere before this call.

Reply:
We always enter DtlsGetHeader with at least 2 bytes (the minimum DTLS header). I added by the way an extra check.